tests/test_ffmpeg.py

```diff
@@ -140,21 +151,34 @@ def test_reader_more():
             count += 1
     assert count == len(R)
     assert count in (35, 36)  # allow one frame off size that we know
-    # Test index error -1
-    raises(IndexError, R.get_data, -1)
+    raises(IndexError, R.get_data, -1) # Test index error -1
+    
     # Now read beyond (simulate broken file)
     with raises(RuntimeError):
-        R._read_frame()  # ffmpeg seems to have an extra frame, avbin not?
+        R._read_frame() # ffmpeg seems to have an extra frame, avbin not?
         R._read_frame()
-    
-    # Test  loop
+
+    # Set the image index to 0 and go again
+    R.set_image_index(0)
+    count2 = 0
+    while True:
+        try:
+            R.get_next_data()
+        except IndexError:
+            break
+        else:
+            count2 += 1
+    assert count2 == count
+    raises(IndexError, R.get_data, -1) # Test index error -1
+
+    # Test loop
     R = imageio.read(get_remote_file('images/realshort.mp4'), 'ffmpeg', loop=1)
     im1 = R.get_next_data()
     for i in range(1, len(R)):
         R.get_next_data()
     im2 = R.get_next_data()
     im3 = R.get_data(0)
-    im4 = R.get_data(2)  # touch skipping frames
+    im4 = R.get_data(2) # touch skipping frames
```

Comment:
The CI fails over some style issues: https://travis-ci.org/imageio/imageio/jobs/282768272#L558
Flake8 want two spaces before an in-line comment.